app/views/themes/dc_repository/layouts/homepage.html.erb

```diff
@@ -0,0 +1,45 @@
+<% # OVERRIDE: Hyrax v3.4.1 - remove background-container-gradient div, remove site-title-container div, add divs and classes for custom styles for dc_repository %>
+
+<% content_for(:navbar) do %>
+  <div class="image-masthead d-flex">
+<!-- dc-repository homepage -->
+    <div class="background-container"  title="<%= block_for(name: 'banner_image_text') %>" style="background-image: url('<%= banner_image %>')"></div>
+    <% # OVERRIDE: Hyrax v3.4.1 - remove background-container-gradient %>
+    <% # OVERRIDE: Hyrax v3.4.1 - remove site-title-container %>
+    <% # OVERRIDE: Hyrax v3.4.1 - add divs and classes for custom styles %>
+    <div class="utk-digital-elements-wrapper">
+      <div class="container-fluid d-flex">
+        <div class='row'>
+          <div class="utk-digital-search-container d-flex">
+            <div class="utk-digital-search-heading d-flex">
+              <span class="utk-heading-1" role="heading" aria-level="1">Digital Collections</span>
+              <span class="utk-description" role="subheading"><span>Explore Items digitized from our collections.</span></span>
+            </div>
+          </div>
+          <div class='circle-container d-flex hidden-sm'>
+            <div class="circle d-flex">
+              <div class='circle-text'>
+                <%# Dynamically show number of collections and works %>
+                <%= number_to_human(Collection.count, units: {thousand: "K"}) %> Collections
+                <% work_count = GenericWork.count + Image.count + Audio.count + Book.count + CompoundObject.count + Newspaper.count + Pdf.count + Video.count %>
+                <%= number_to_human(work_count, units: {thousand: "K"}) %> Works and more.
```

Comment:
is there a way for the "K" to be next to the number, without a space? e.g. "14.4K" instead of "14.4 K"?

I didn't reference the wireframe, but I feel like using the "k" to denote thousands normally doesn't have a space. something about it looks off. especially when the K goes to the next line like in one of the screenshots.

Reply:
This has been fixed with this commit https://github.com/scientist-softserv/utk-hyku/pull/122/commits/a22637a70fbe99167f05242f6613b8ededdcccc9

![Screenshot 2022-11-08 at 9 52 53 AM](https://user-images.githubusercontent.com/39319859/200640284-d899c458-04a5-4f86-9e87-f0bdd115f626.JPG)